pkg/volume/volume.go

```diff
@@ -55,10 +71,6 @@ func MountVolume(devicePath, mountPath, fsType string, mountOptions []string) er
 		}
 	}
 
-	if fsType == "" {
```

Comment:
Can this input be empty now? If YES, we may need to retain this fallback/default value.

Reply:
Already handled as part of the kubernetes mounter.